core/src/main/java/pl/agh/edu/ui/window/DifficultyWindow.java

```diff
@@ -0,0 +1,165 @@
+package pl.agh.edu.ui.window;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.badlogic.gdx.scenes.scene2d.ui.ButtonGroup;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.NinePatchDrawable;
+
+import pl.agh.edu.config.GraphicConfig;
+import pl.agh.edu.engine.hotel.dificulty.DifficultyLevel;
+import pl.agh.edu.ui.GameSkin;
+import pl.agh.edu.ui.component.button.DifficultyButton;
+import pl.agh.edu.ui.component.label.LanguageLabel;
+import pl.agh.edu.ui.language.LanguageManager;
+import pl.agh.edu.ui.resolution.Size;
+import pl.agh.edu.ui.utils.FontType;
+import pl.agh.edu.ui.utils.SkinColor;
+import pl.agh.edu.ui.utils.wrapper.WrapperContainer;
+
+public class DifficultyWindow extends WrapperContainer<Table> {
+	public final GameSkin skin = GameSkin.getInstance();
+	public final Table frame = new Table();
+	public final List<DifficultyButton> buttonsList = new ArrayList<>();
+	public final ButtonGroup<TextButton> buttonGroup = new ButtonGroup<>();
+	private LanguageLabel titleLabel;
+	private TextButton backButton;
+	private TextButton playButton;
+	private int width;
+	private int height;
+	private float largePaddingMultiplier = 1;
+
+	public DifficultyWindow() {
+		setActor(frame);
+		getSize();
+		createDifficultyButtons();
+		createTitleLabel();
+		createPlayButton();
+		createBackButton();
+		createFrame();
+		LanguageManager.addListener(this);
+	}
+
+	public void createFrame() {
+		frame.clearChildren();
+		frame.left().top();
+		frame.setFillParent(true);
+		frame.background(skin.getDrawable("hotel-room"));
+		frame.add(createTitleLabelTable()).left().padLeft(width / 12).padTop(largePaddingMultiplier * height / 16).expandX().row();
+		addDifficultyButtonsToFrame();
+		addPlayBackButtonsToFrame();
+	}
+
+	private Table createTitleLabelTable() {
+		Table titleLabelTable = new Table();
+		titleLabelTable.setBackground(getTitleLabelBackground());
+		titleLabelTable.add(titleLabel).pad(10f, 40f, 10f, 40f);
+		return titleLabelTable;
+	}
+
+	private void addPlayBackButtonsToFrame() {
+		Table playBack = new Table();
+		playBack.add(backButton).padRight(3 * width / 5);
+		playBack.add(playButton);
+		frame.add(playBack).padTop(largePaddingMultiplier * height / 15);
+	}
+
+	private void addDifficultyButtonsToFrame() {
+		for (DifficultyButton button : buttonsList) {
+			frame.add(button).left().padTop((int) (largePaddingMultiplier * height / 24)).padLeft(width / 6 + buttonsList.indexOf(button) * width / 24).row();
+		}
+	}
+
+	public void createDifficultyButtons() {
+		for (DifficultyLevel level : DifficultyLevel.values()) {
```

Comment:
można to zapisać przy użyciu foreach. coś w wstylu:
`Arrays.stream(DifficultyLevel.values()).forEach(level -> buttonsList.add(new DifficultyButton(level)))`
podobnie niżej